inference/core/utils/image_utils.py

```diff
@@ -98,29 +99,33 @@ def load_image_inferred(value: Any) -> Image.Image:
 pattern = re.compile(r"^data:image\/[a-z]+;base64,")
 
 
-def load_image_base64(value):
-    """Loads an image from a base64 encoded string.
+def load_image_base64(value: str) -> np.ndarray:
+    """Loads an image from a base64 encoded string using OpenCV.
 
     Args:
         value (str): Base64 encoded string representing the image.
 
     Returns:
-        Image.Image: The loaded PIL image.
+        np.ndarray: The loaded image as a numpy array.
     """
     # New routes accept images via json body (str), legacy routes accept bytes which need to be decoded as strings
     if not isinstance(value, str):
         value = value.decode("utf-8")
+
     try:
         value = base64.b64decode(value)
```

Comment:
We may want to look into `pybase64` which is supposed to be many times faster for decoding: https://github.com/mayeut/pybase64

Reply:
Yes! Will definitely test this when I get to testing the http interface (so far all testing is in python script land)